### PR TITLE
Add condition for Windows executable in Launcher::launch().

### PR DESCRIPTION
**Pull request by hongaar**
Add support for Windows.

Don't know if this would be something that can be tested in CI, but confirmed working on my Windows 10 machine.

### CODE REVIEW

File: src/ChromeDevtoolsProtocol/Instance/Launcher.php [hunk 1]
```diff
@@ -113,6 +114,13 @@ public function launch(ContextInterface $ctx, ...$args): ProcessInstance
 				throw new RuntimeException(sprintf("Executable [%s] not found.", static::DEFAULT_LINUX_EXECUTABLE));
 			}
 
+		} else if (stristr(PHP_OS, 'WIN')) {
```
Comment (jakubkulhan) [inline src/ChromeDevtoolsProtocol/Instance/Launcher.php]: - Please use `"` for strings.
- `stristr("Darwin", "WIN") === 3` => breaks OS X. `PHP_OS` string imo should _start_ with `Win`, so it should be `strncasecmp(PHP_OS, "Win", 3) === 0`

File: src/ChromeDevtoolsProtocol/Instance/Launcher.php [hunk 2]
```diff
@@ -113,6 +114,13 @@ public function launch(ContextInterface $ctx, ...$args): ProcessInstance
...
+			$finder = new ExecutableFinder();
+			$executable = $finder->find(static::DEFAULT_WINDOWS_EXECUTABLE);
+			if ($executable === null) {
+    			throw new RuntimeException(sprintf("Executable [%s] not found.", static::DEFAULT_WINDOWS_EXECUTABLE));
```
Comment (jakubkulhan) [inline src/ChromeDevtoolsProtocol/Instance/Launcher.php]: Should be indented.

**Review by jakubkulhan — CHANGES_REQUESTED** [review]
Thank you for your PR! I've added some comments.

File: src/ChromeDevtoolsProtocol/Instance/Launcher.php (hunk 1 above)
Comment (hongaar) [reply src/ChromeDevtoolsProtocol/Instance/Launcher.php]: Good catch, copied detection where OS X case would be above Windows.

File: src/ChromeDevtoolsProtocol/Instance/Launcher.php (hunk 2 above)
Comment (hongaar) [reply src/ChromeDevtoolsProtocol/Instance/Launcher.php]: Was a tabs/spaces issue, now fixed.

**Comment by hongaar** [comment]
Thanks for your review. Requested changes are in.

**Review by jakubkulhan — APPROVED** [review]

**Comment by jakubkulhan** [comment]
Thank you for all your work!

**Comment by hongaar** [comment]
Nothing compared to the amount of work and love this library already got. Glad to help of course!

**Pull request merged**